qucs-core/src/components/tswitch.cpp

```diff
@@ -22,6 +22,15 @@
  *
  */
 
+/*!
+ * \file tswitch.cpp
+ * \brief time controlled switch class implementation
+ *
+ * \todo add a property to allow choosing the resistance profile
```

Comment:
yes, the `\todo` should have been removed... the latest commit implemented exactly that...

There are others `\todo` around that should be checked, I think. Will address this when doing the Big Formatting Cleanup, one day :grin: